cloud/amazon/ec2_ami_find.py

```diff
@@ -352,12 +366,16 @@ def main():
         filter['architecture'] = architecture
     if hypervisor:
         filter['hypervisor'] = hypervisor
+    if owner_alias:
+        filter['owner_alias'] = owner_alias
```

Comment:
I believe the owner-alias filter is already covered by the `owner` argument - see next comment.